src/lib/modules/response/resp_delay.go

```diff
@@ -46,10 +54,11 @@ func HandleGetRequest(url string) int {
 		// Prone to alerting, printing for now
 		fmt.Println(err)
 	}
+	len := resp.ContentLength
```

Comment:
`len` is a keyword in golang. Please change this to any other relevant name. Thanks

Reply:
Yup, thanks!